docs/src/main/sphinx/admin/properties.rst

```diff
@@ -21,3 +21,80 @@ may be used to tune Trino or alter its behavior when required.
     Logging <properties-logging>
     Web UI <properties-web-interface>
     Regular expression function <properties-regexp-function>
+
+Property value types
+--------------------
+
+Configuration properties in Trino take values of different types, some standard
+and some non-standard. Supported values may be further limited on a per-property
```

Comment:
there is no standard vs non standard ... 

they are all strings in .properties files .. and Trino supports the following parsing to support different data types

Reply:
I agree with @mosabua . Saying "non-standard" doesn't add much value, but suggests something is wrong and could be improved.

Reply:
Reworded a bit.